### PR TITLE
Spanish lesson 4 store data on smart contract

### PR DESCRIPTION
**Pull request by marianar97**
This PR also includes lesson 3.

### CODE REVIEW

**Comment by adilanchian** [comment]
@marianar97 - can you list the lessons here that are included please? Just ease of use for others to see!

File: Solidity_And_Smart_Contracts/es/Section_1/Leccion_3_Compilar_Contracto_Localmente.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Entonces, incluso cuando trabajamos localmente, queremos imitar ese entorno. Técnicamente, podríamos simplemente compilar y ejecutar el código de Solidity, pero lo que hace que Solidity sea mágico es cómo puede interactuar con las billeteras de blockchain y Ethereum (de lo que veremos más en la próxima lección). Entonces, es mejor simplemente hacer esto de una vez.
+
+Vamos a escribir un script personalizado que haga esos tres pasos por nosotros.
+
+¡Vamos a hacerlo! 
+
+📝 Crear un script para correr nuestro contrato
+-------------------------------------
+
+Ve al directorio ** `scripts` ** y crea un archivo llamado **` run.js`. **
```
[the remark this answers is not in the document]
Comment (marianar97) [reply Solidity_And_Smart_Contracts/es/Section_1/Leccion_3_Compilar_Contracto_Localmente.md:22]: done

**Comment by adilanchian** [comment]
Adding some comment here about filename :)

**Comment by marianar97** [comment]
Hey, I changed the filenames. I see there are some merge conflicts, lmk if I can help

**Comment by adilanchian** [comment]
Hey @marianar97 -

Sorry for the late reply here!!

Ya if you can fix these merge conflicts, that would be epic!

**Pull request closed without merging**